node/actors/bft/src/testonly/twins/partition.rs

```diff
@@ -0,0 +1,115 @@
+use std::cmp::min;
+
+/// A group of nodes that can talk to each other.
+pub type Partition<'a, T> = Vec<&'a T>;
```

Comment:
is Partition/Partitioning naming from the paper? Isn't https://en.wikipedia.org/wiki/Partition_of_a_set a more standard naming? I.e. Partion -> Set, Partitioning -> Partition?

Reply:
if you want to keep this naming, then please rename partitions to partitionings consistently

Reply:
Renamed to `Split` and `splits`